src/saveReportAsJson.js

```diff
@@ -0,0 +1,9 @@
+import fs from 'fs';
+import { shell } from 'electron';
+
+export default async function makeJSON(data, savePath) {
+  fs.writeFile(savePath, data, (error) => {
+    if (error) throw error;
+    return shell.openPath(savePath);
```

Comment:
Opening `.json` by default will in most cases just launch the text editor, raw json isn't something people generally would want to look at other than to verify that it did get saved.

I feel we should do one of these:
1. Open the folder where it's saved.
2. Show a toast saying that the file was saved at whatever path and clicking the toast opens the folder.

I don't know if we have UI for a toast, if we don't you can go with 1.

Reply:
Totaly :100: 

Have identified a way to add toast too, but for now opening the directly seems like a good thing to do.